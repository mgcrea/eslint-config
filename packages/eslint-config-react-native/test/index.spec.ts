import { describe, expect, it } from "vitest";
import * as module from "./..";

describe("module", () => {
  it("should export an eslint config", () => {
    expect(module).toMatchSnapshot();
  });
});
