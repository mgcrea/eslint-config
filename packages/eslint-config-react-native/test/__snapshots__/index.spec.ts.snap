// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`module > should export an eslint config 1`] = `
{
  "default": {
    "extends": [
      "eslint:recommended",
      "plugin:@typescript-eslint/recommended",
      "plugin:react/recommended",
      "plugin:react-hooks/recommended",
      "plugin:prettier/recommended",
      "plugin:testing-library/react",
    ],
    "globals": {
      "__DEV__": true,
      "module": false,
      "process": false,
      "require": false,
    },
    "parser": "@typescript-eslint/parser",
    "plugins": [
      "@typescript-eslint",
      "@react-native",
      "react-native",
    ],
    "rules": {
      "@typescript-eslint/no-unused-vars": [
        "warn",
        {
          "argsIgnorePattern": "^_",
          "ignoreRestSiblings": true,
        },
      ],
      "react/react-in-jsx-scope": "off",
    },
    "settings": {
      "react": {
        "version": "detect",
      },
    },
  },
}
`;
